easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb

```
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'LASSO-Python'
+version = '2.0.0'
+
+homepage = 'https://open-lasso-python.github.io/lasso-python/'
+description = """This python library is designed for general purpose usage in
+the field of Computer Aided Engineering (CAE). It's name originates from the
+original initiator and donator of the project Lasso GmbH. The library is now
+maintained by an open-source community."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('h5py', '3.8.0'),
+    ('SciPy-bundle', '2023.02'),
+    ('plotly.py', '5.13.1'),
+    ('matplotlib', '3.7.0'),
+    ('scikit-learn', '1.2.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('mdurl', '0.1.2', {
+        'checksum': 'bb413d29f5eea38f31dd4754dd7377d4465116fb207585f97bf925588687c1ba',
+    }),
+    ('markdown-it-py', '2.2.0', {
+        'modulename': 'markdown_it', 
```

remove trailing whitespace
```suggestion
        'modulename': 'markdown_it',
```